src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/ConfigurationBinder/GetValue.generated.txt

```diff
@@ -64,19 +59,12 @@ namespace Microsoft.Extensions.Configuration.Binder.SourceGeneration
                 return ParseCultureInfo(value, () => section.Path);
             }
 
-            return null;
-        }
-
-        public static int ParseInt(string value, Func<string?> getPath)
-        {
-            try
-            {
-                return int.Parse(value, NumberStyles.Integer, CultureInfo.InvariantCulture);
-            }
-            catch (Exception exception)
+            if (type == typeof(int))
```

Comment:
ints are pretty common in config. It might make sense to keep this near the top of the check, so we don't have to check a bunch of other (less common) types first.